### ISSUE TITLE
Cannot resolve Microsoft.FSharp.Core.LanguagePrimitives.FloatWithMeasure

### ISSUE THREAD

**Issue by anchann** [fable-compiler/Fable#1979]
### Description

Getting an error `FABLE: Cannot resolve Microsoft.FSharp.Core.LanguagePrimitives.FloatWithMeasure` when trying to compile code.

### Repro code

```
open Fable.Core
open FSharp.Data.UnitSystems.SI.UnitSymbols

let distance: float<m> = LanguagePrimitives.FloatWithMeasure 0.0
Fable.Core.JS.console.log distance
```

### Expected and actual results

Code compiles.

### Related information

* Fable version: 2.4.14, also https://fable.io/repl/
* Operating system: MacOS 10.13.6

**Comment by MangelMaxime** [comment]
Released in version fable-compiler 2.4.15

The REPL released should follow when I have fixed another bug specific to the REPL

**Comment by anchann** [comment]
Thank you @MangelMaxime !
